### PR TITLE
Feature: Integrate mock service into runtimes

### PR DESCRIPTION
**Pull request by MP91**
This PR integrates Kuksa.Val's mock service into all Velocitas runtimes.

### CODE REVIEW

File: mock.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
+# /********************************************************************************
```
Comment (dennismeister93) [inline mock.py]: should this license header fit to "our" velocitas header?

File: runtime-k3d/src/runtime/deployment/gen_helm.py [hunk 2]
```diff
@@ -78,6 +78,13 @@ def generate_values_by_service(service: Service) -> dict[str, Any]:
     if service.config.ports:
         value_spec[value_spec_key]["ports"] = generate_ports_spec(service.config)
 
+    value_spec[value_spec_key]["mounts"] = list()
+    for mount in service.config.mounts:
+        from_to = mount.split(":")
+        value_spec[value_spec_key]["mounts"].append(
+            {"id": "foo", "from": from_to[0], "to": from_to[1]}
```
Comment (dennismeister93) [inline runtime-k3d/src/runtime/deployment/gen_helm.py]: id:  foo?

File: runtime-kanto/src/runtime/runtime_kanto.py [hunk 3]
```diff
@@ -79,6 +83,19 @@ def adapt_feedercan_deployment_file():
         json.dump(data, f, indent=4)
 
 
+def adapt_mockservice_deployment_file():
```
Comment (dennismeister93) [inline runtime-kanto/src/runtime/runtime_kanto.py:86]: lets maybe have a generic function for adapting deployment files

**Review by dennismeister93 — CHANGES_REQUESTED** [review]

File: mock.py (hunk 1 above)
Comment (doosuu) [reply mock.py]: Done

File: runtime-k3d/src/runtime/deployment/gen_helm.py (hunk 2 above)
Comment (doosuu) [reply runtime-k3d/src/runtime/deployment/gen_helm.py]: Removed. `id` is unused at the moment

File: runtime-kanto/src/runtime/runtime_kanto.py (hunk 3 above)
Comment (doosuu) [reply runtime-kanto/src/runtime/runtime_kanto.py:86]: Will be tackled with the auto-generation of Kanto deployment files

**Review by dennismeister93 — APPROVED** [review]
lgtm

**Pull request merged**